src/main/java/seedu/address/logic/parser/quiz/QuizAnswerParser.java

```diff
@@ -14,14 +18,17 @@
     /**
      * Used for initial separation of question index and answer.
      */
-    private static final Pattern BASIC_INPUT_FORMAT = Pattern.compile("(?<index>\\s+)(?<answer>)");
+    private static final Pattern BASIC_INPUT_FORMAT = Pattern.compile("(?<index>\\S+)(?<answer>.*)");
 
     /**
      * Parses the given {@code String} of user input in the context of the quiz answer
      * and returns an QuizCheckAnswer object for execution.
      */
-    public QuizCheckAnswer parse(String userInput) {
+    public QuizCheckAnswer parse(String userInput) throws ParseException {
         final Matcher matcher = BASIC_INPUT_FORMAT.matcher(userInput.trim());
+        if (!matcher.matches()) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE));
```

Comment:
Don't just use a generic command and help. Write an error message and help specifically for the answer.

Reply:
Thank you very much for your help! I will fix all problems raised above.